packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -129,7 +129,9 @@ export default class BackupNg {
                     isMatchingVm(obj) &&
                     // don't match replicated VMs created by this very job otherwise
                     // they will be replicated again and again
-                    !('start' in obj.blockedOperations && obj.other['xo:backup:job'] === job.id)
+                    !('start' in obj.blockedOperations && obj.other['xo:backup:job'] === job.id) &&
+                    // don't match temporary VM, like those created by health check
+                    !obj.tags.includes('xo:noBak')
```

Comment:
After discussing with @olivierlambert.

Use kebab-case: `xo:no-bak` to be inline with [VM sync hook](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/xapi/docs/vm-sync-hook.md).

Accept an optional description/explanation after an equal sign:
- `xo:no-bak=Incremental Replication`
- `xo:no-bak=Health Check`

In the future it will be displayed nicely with the introduction of scoped tags.

Reply:
To be done in the UI:

- in the job edition, on the tags section of the smart mode: add a message explaining that VMs with this tag will be ignored.
- in the `Tags` component, add a tooltip for these tags explaining that this VM will be ignored during smart mode backups.

Reply:
IMHO we should do a separate commit with a proper CHANGELOG entry for this.

Reply:
I will create a card in the internal board for the `Tags` component.

Please address the last comment and do a separate commit with a CHANGELOG entry for the `xo:no-bak` tag.